sdk/core/azure-core/test/ut/proxy_tests/builddocker.ps1

```diff
@@ -0,0 +1,16 @@
+pushd .\localproxy
```

Comment:
nit: Add license header to this file just like `Start-WebSocketServer.ps1`.

Reply:
Added.

Reply:
@danieljurek Do we have a backlog item to add checks to validate that the license header is present in all files?